guides/doc-Administering_Red_Hat_Satellite/topics/proc_integrating-satellite-with-red-hat-single-sign-on-for-external-authentication.adoc

```diff
@@ -82,39 +71,50 @@ For more information about audience support, see https://access.redhat.com/docum
 +
 * From the *Mapper Type* list, select *Group Membership*.
 +
-* From the *Token Claim Name* list, select *groups*.
+* In *Token Claim Name*, enter *groups*.
 +
 * Set the *Full group path* toggle to *OFF*.
 +
 For more information about group mappers, see https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.3/html/server_administration_guide/user-storage-federation#ldap_mappers[Group Mapper] in the _LDAP Mappers_ section of the _{Keycloak} Server Administration Guide_.
 
 .. Click *Save*.
-
-. In the {Project} web UI, navigate to *Administer* > *Settings*, and click the *Authentication* tab.
+.. Navigate to *Groups*, and click *New*.
++
+* Enter a *Name* for the group.
+* Click *Save*.
+.. Navigate to *Users* and click *Add user*. 
++
+* Enter the *Username*, *Email*, and other user details. Set the *User Enabled* toggle to *ON*.
++
+* Click *Save*. The user is now created.
++
+* Click the *Credentials* tab and reset the user password if required. 
```

Comment:
nit: Resetting the password is not a optional step, passwords needs to be set and the `Temporary` must be set to off.

Reply:
Fixed, thanks.